lib/Conversion/SeqToSV/FirRegLowering.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "FirRegLowering.h"
 #include "circt/Dialect/Comb/CombOps.h"
+#include "circt/Dialect/FIRRTL/FIRRTLUtils.h"
```

Comment:
Is this an okay dependency? Do these conversions already depend on the FIRRTL dialect? If not, there may be places that need to be updated for that.

Reply:
I included this header for the parallel transform/reduce utility defined there, not for anything related to the FIRRTL dialect. I've moved the transform/reduce helper to a common circt utility header and included that instead.